pkg/kubernetes/scripts/kube-client-cockpit.js

```diff
@@ -891,7 +891,10 @@
                     return defer.promise;
 
                 var last, req, kubectl, loginOptions;
-                var loginData = window.localStorage.getItem('login-data');
+                var loginData = window.localStorage.getItem(cockpit.transport.application + ':login-data');
```

Comment:
Perhaps accept a storage so it can operate on either localStorage or sessionStorage?